app/views/devise/sessions/new.html.slim

```diff
@@ -0,0 +1,19 @@
+- content_for(:description) do
+  h2 = t('devise.sessions.description')
+
+= form_with(model: resource, url: session_path(resource_name), local: true, html: { class: 'mt-8' }) do |f|
+  .form-group
+    = f.label :email
+    = f.email_field :email, autofocus: true, autocomplete: 'email', class: 'form-control form-control-full'
+
+  .form-group
+    = f.label :password
+    .relative.grid.items-center
+      = f.password_field :password, autocomplete: 'current-password', class: 'form-control form-control-full'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/b2f39155e54d78bea577b04a53a9496122f58f84/app/views/devise/sessions/new.html.slim#L12"></span>Line is too long. [109/80]</td>
    </tr>
  </tbody>
</table>